x-pack/test/detection_engine_api_integration/security_and_spaces/tests/update_rules_bulk.ts

```diff
@@ -94,6 +95,64 @@ export default ({ getService }: FtrProviderContext) => {
         expect(bodyToCompare2).to.eql(outputRule2);
       });
 
+      it('should update two rule properties of name using the two rules rule_id and migrate actions', async () => {
+        const [connector, rule1, rule2] = await Promise.all([
+          supertest
+            .post(`/api/actions/connector`)
+            .set('kbn-xsrf', 'foo')
+            .send({
+              name: 'My action',
+              connector_type_id: '.slack',
+              secrets: {
+                webhookUrl: 'http://localhost:1234',
+              },
+            }),
+          createRule(supertest, getSimpleRule('rule-1')),
+          createRule(supertest, getSimpleRule('rule-2')),
+        ]);
+        await Promise.all([
+          createLegacyRuleAction(supertest, rule1.id, connector.body.id),
+          createLegacyRuleAction(supertest, rule2.id, connector.body.id),
+        ]);
+
+        expect(rule1.actions).to.eql([]);
+        expect(rule2.actions).to.eql([]);
+
+        const updatedRule1 = getSimpleRuleUpdate('rule-1');
+        updatedRule1.name = 'some other name';
+
+        const updatedRule2 = getSimpleRuleUpdate('rule-2');
+        updatedRule2.name = 'some other name';
+
+        // update both rule names
+        const { body } = await supertest
+          .put(`${DETECTION_ENGINE_RULES_URL}/_bulk_update`)
+          .set('kbn-xsrf', 'true')
+          .send([updatedRule1, updatedRule2])
+          .expect(200);
+
+        // @ts-expect-error
```

Comment:
You should just type body above from `any` into what the `_bulk_update` is returning and then you can remove this `ts-expct-error`. We really don't want to add more of them to the code base when we can fix them by casting or using types.